### PR TITLE
General - Silver Content Videos

### PR DESCRIPTION
**Pull request by jessechoe10**

### CODE REVIEW

File: content/1_General/Learning_to_Code.mdx [hunk 1]
```diff
@@ -32,6 +32,15 @@ Please let us know what works (or doesn't) for you!
 	<Resource title="w3schools" url="https://www.w3schools.com/">
 		Free tutorials for C++, Java, Python 3.
 	</Resource>
+	<Resource title="YouTube" url="https://www.youtube.com/watch?v=VCnuPywb9iU&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=2">
```
Comment (HarryWangATX) [inline content/1_General/Learning_to_Code.mdx:35]: one resource for all languages is fine

File: content/2_Bronze/Intro_Graphs.mdx [hunk 2]
```diff
@@ -6,6 +6,12 @@ description: 'Visualizing problems as graphs with resources covering basic termi
 frequency: 3
 ---
 
+<Resources>
+	<Resource source = "YouTube" title = "Graph Representations" url = "https://www.youtube.com/watch?v=XZGsBUgmeXc&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=4">
+		A video that explains what graphs are, the different types of graphs, and the different ways to represent them using code. He briefly describes how to represent these graphs in C++ as well as how they will be represented in USACO problems.
```
Comment (HarryWangATX) [inline content/2_Bronze/Intro_Graphs.mdx:11]: Just say like intro to graphs or smth short

File: content/2_Bronze/Intro_Sorting.mdx [hunk 3]
```diff
@@ -8,6 +8,12 @@ prerequisites:
   - intro-ds
 ---
 
+<Resources>
+	<Resource source = "YouTube" title = "Sorting and Searching" url = "https://www.youtube.com/watch?v=ZhDWGaVJdOw&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=12">
+		A video source that demonstrates how to sort an array in C++, Java, and Python. Start at time 0:49 and end at 5:42.
```
Comment (HarryWangATX) [inline content/2_Bronze/Intro_Sorting.mdx:13]: demonstrates how to sort arrays

File: content/2_Bronze/Time_Comp.mdx [hunk 4]
```diff
@@ -18,6 +18,9 @@ description: "Measuring the number of operations an algorithm performs."
 	<Resource source="Youtube" title="Introduction to Big-O" url="https://www.youtube.com/watch?v=zUUkiEllHG0">
 		If you prefer watching a video instead
 	</Resource>
+	<Resource source="YouTube" title = "Introduction to Algorithms" url="https://www.youtube.com/watch?v=-dOcsV7pmzQ&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=1">
+		Another video source to help you understand Time Complexity with respect to USACO. Start at time 5:58 and end at time 9:04.
+	</Resource>
```
Comment (HarryWangATX) [inline content/2_Bronze/Time_Comp.mdx:23]: is this necessary?

File: content/3_Silver/2P.mdx [hunk 5]
```diff
@@ -150,6 +150,9 @@ export const problems = {
 	>
 		video explanation of two pointers
 	</Resource>
+	<Resource source = "YouTube" title = "Two-Pointer Technique" url = "https://www.youtube.com/watch?v=gLfVHfe_DD0&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=17">
+		A video explanation of common problems that the two-pointer technique could be applied to. Has a bit of Python code but mostly consists of the intuition and problem-solving approach for these problems.
```
Comment (HarryWangATX) [inline content/3_Silver/2P.mdx:154]: video tutorial for two pointers

File: content/3_Silver/Binary_Search.mdx [hunk 6]
```diff
@@ -224,6 +224,9 @@ export const problems = {
 	<Resource source="TC" title="Binary Search" url="binary-search">
 		similar material
 	</Resource>
+	<Resource source="YouTube" title="Sorting and Searching" url = "https://www.youtube.com/watch?v=ZhDWGaVJdOw&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=12">
+		A video explanation to Binary Search and the internal C++ library functions. Begin at time 5:50 and end at 10:19.
```
Comment (HarryWangATX) [inline content/3_Silver/Binary_Search.mdx:228]: video tutorial for binary search

File: content/3_Silver/DFS.mdx [hunk 7]
```diff
@@ -187,6 +187,9 @@ not a good example
 	<Resource source="CPH" title="12.1 - DFS">
 		example diagram + code
 	</Resource>
+	<Resource source = "YouTube" title = "DFS" url = "https://www.youtube.com/watch?v=-aDJhOAwI1I&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=6">
+		A video explaining the intuition behind DFS as well as the iterative and recursive implementation of DFS.
```
Comment (HarryWangATX) [inline content/3_Silver/DFS.mdx:191]: dfs video explanation

File: content/3_Silver/Greedy_Sorting.mdx [hunk 8]
```diff
@@ -186,6 +186,9 @@ export const problems = {
 	>
 		slides from Intro to Algorithms
 	</Resource>
+	<Resource source = "YouTube" title = "Greedy Algorithms" url = "https://www.youtube.com/watch?v=vOYDh_st-K8&list=PLtLxhThTxJMVMvWJoE2p8Z84tA8suGzvn&index=15">
+		A video demonstrating classic greedy problems such as the scheduling problem. He codes the solutions to some of the problems in Python, while mostly explaining the intuition and problem-solving strategy behind the solution.
```
Comment (HarryWangATX) [inline content/3_Silver/Greedy_Sorting.mdx:190]: video demonstration of greedy

File: content/3_Silver/Sorting_Custom.mdx [hunk 9]
```diff
@@ -68,21 +68,21 @@ export const problems = {
 			['Sorting', 'Prefix Sums'],
 			'usaco-645'
 		),
-		new Problem('Silver', 'Triangles', '1015', 'Hard', false, ['Sorting'], ''),
 		new Problem(
 			'Silver',
 			'Out Of Sorts',
 			'834',
-			'Hard',
+			'Normal',
```
Comment (HarryWangATX) [inline content/3_Silver/Sorting_Custom.mdx:75]: ???

File: content/3_Silver/Sorting_Custom.mdx [hunk 10]
```diff
@@ -68,21 +68,21 @@ export const problems = {
...
 			['Sorting'],
 			''
 		),
+		new Problem('Silver', 'Triangles', '1015', 'Hard', false, ['Sorting'], ''),
 		new Problem(
 			'Silver',
 			'Meetings',
 			'967',
-			'Very Hard',
+			'Hard',
```
Comment (HarryWangATX) [inline content/3_Silver/Sorting_Custom.mdx:85]: ???

**Review by HarryWangATX — CHANGES_REQUESTED** [review]

**Comment by HarryWangATX** [comment]
i havent watch the videos yet

**Comment by thecodingwizard** [comment]
Oops I personally think we can find better videos (I think williamfiset videos or others are of a higher quality). But if others think the videos are fine then sure :)

**Comment by HarryWangATX** [comment]
> Oops I personally think we can find better videos (I think williamfiset videos or others are of a higher quality). But if others think the videos are fine then sure :)

yea i just took a look i feel like those vids are a little un serious xD

**Comment by jessechoe10** [comment]
> > Oops I personally think we can find better videos (I think williamfiset videos or others are of a higher quality). But if others think the videos are fine then sure :)
> 
> yea i just took a look i feel like those vids are a little un serious xD

Sure. I would agree. However, I think adding some humor makes watching much more interesting. Also, he has a lot of USACO problem walkthroughs so could I add his vids for those?

**Comment by HarryWangATX** [comment]
i think usaco editorials are well written for all their problems

**Comment by nwatx** [comment]
> 
> 
> > > Oops I personally think we can find better videos (I think williamfiset videos or others are of a higher quality). But if others think the videos are fine then sure :)
> > 
> > 
> > yea i just took a look i feel like those vids are a little un serious xD
> 
> Sure. I would agree. However, I think adding some humor makes watching much more interesting. Also, he has a lot of USACO problem walkthroughs so could I add his vids for those?

personally I find that humor to be quite distracting from the essence of the video

File: content/3_Silver/Sorting_Custom.mdx (hunk 10 above)
Comment (frodakcin) [reply content/3_Silver/Sorting_Custom.mdx:85]: We told him to do this in the other PR

File: content/3_Silver/Sorting_Custom.mdx (hunk 9 above)
Comment (frodakcin) [reply content/3_Silver/Sorting_Custom.mdx:75]: idk how hard out of sorts is

File: content/3_Silver/Sorting_Custom.mdx (hunk 10 above)
Comment (HarryWangATX) [reply content/3_Silver/Sorting_Custom.mdx:85]: Ah

**Pull request closed without merging**